cloudsensei/agg_lambda/lambda_function.py

```diff
@@ -28,7 +28,7 @@ def lambda_handler(event, context):
         index_id = f"{account}-{current_date}"
         es_data = es_operations.get_es_data_by_id(index_id)
         if es_data:
-            email_body += f"<h2>{es_data.get('_source', {}).get('subject')}</h2>"
+            email_body += f"<h2>Cloud Report: Long running instances in the @{account} account</h2>"
```

Comment:
This should be `Weekly Cloud Report: Long running instances in the Perf&Scale AWS Accounts`
Per line https://github.com/redhat-performance/cloud-governance/blob/main/cloudsensei/agg_lambda/lambda_function.py#L25

Reply:
Mentioned in the Subject, I think no need for every account.

Reply:
Will this be a daily or weekly report?

Reply:
weekly report.

Reply:
Cool, please add weekly, to help users distinguish it.